classes/02class/exercises/c02-network01/starkmatt/c02-network01.txt

```diff
@@ -0,0 +1,67 @@
+<h2>Execrise 1</h2>
+
+#10.0.0.1/24
+
+IP Address in Binary:00001010.00000000.00000000.00000001
+
+Subnet in Binary:11111111.11111111.11111111.00000000
+
+Subnet in Decimals:255.255.255.0
+
+Private / Public: Private 
+
+IP Addresses for Hosts available (AWS): 251
+
+#192.168.0.1/16
+
+IP Address in Binary: 11000000.10101000.00000000.00000001
+
+Subnet in Binary: 11111111.11111111.00000000.00000000
+
+Subnet in Decimals:255.255.0.0
+
+Private / Public: Private
+
+IP Addresses for hosts available (AWS): 65,534
+
+
+#249.165.166.135/30
+
+IP Address in Binary: 11111001.10100101.10100110.10000111
+
+Subnet in Binary: 11111111.11111111.11111111.11111100
+
+Subnet in Decimals: 255.255.255.252
+
+Private / Public: Private
+
+IP Addresses for hosts available (AWS):
+2 (Unsure if my calculations were correct)
```

Comment:
that would be 4, as we have 2^2 = 4. But in AWS, the minimum subnet size is /28, so a /30 subnet is not valid in AWS.

Reply:
actually just 2 valid hosts, 4 ip counts less 2 (network + broadcast) but as stated by Denis, AWS required minimum /28 as they need 5 ips